### ISSUE TITLE
Crimea is reported as the Russian territory

### ISSUE THREAD

**Issue by asiryk-888** [nvkelso/natural-earth-vector#838]
ne_50m_admin_0_countries map reports Crimea peninsula as the territory of the Russian Federation, not Ukraine. Could you, please, make the correction?

**Comment by GutmannCH** [comment]
I have no legitimity on NE data, but the disclaimer looks clear: "Natural Earth shows de facto boundaries by default according to who controls the territory, versus de jure.", please refer to https://www.naturalearthdata.com/about/disputed-boundaries-policy/
Hence the RUC code as a distinct sub unit code, as de facto RU more than UA.
But I would agree that, even going down at the NE sub map unit level, Crimea is still listed as ISO -99, but it should be UA (UA-43 Avtonomna Respublika Krym) according to ISO (https://www.iso.org/obp/ui/#iso:code:3166:UA).
De facto administered by Russia but not according to the rest of the world (NE data ADM0_A3_XX look correct in this case at the sub unit level).

**Comment by asiryk-888** [comment]
Well, but for this case the approach must be different as Russia has no fixed borders and in line with the logic you are trying to explain also Kherson, Zaporizhzhya, Donetsk and Luhansk areas of Ukraine are largely controlled by the Russian combatants now (formally, Russia applied the same approach as for Crimea there and according to the constitution they belong to the failed state). The "now" would significantly differ from hour-to-hour. Therefore, it would make a suggestion to bring Crimea back to Ukraine until it is all settled to not violate the logic and, which is more important, the international agreements / law.

**Comment by GutmannCH** [comment]
BTW please refer to #810 and #812 , lengthy discussions on the subject "de facto" vs. "de jure".
I am not very knowledgeable on NE (yet :) ) but I would vote for a review of the ISO codification/recognition of these disputed territories in the various POV maps. There may be consequences and impacts I am not aware of, on the NE internal machinery that would explain inconsistencies...

**Comment by RonaldCalitri** [comment]
There are several disputed territories - The most powerful are not always internationally recognized to control territories they claim Ukraine - recognized as such by everybody> Rule of International Law please a "country" must be internationally recognized, - Morocco would be a less (but still) controversial case. 
and Israel ? The Golan Heights? I don't want to know, but hopefully not penciled in just because the US (and only the US) has recognized but _everybody else shuddered.

**Comment by megamster** [comment]
> Well, but for this case the approach must be different as Russia has no fixed borders and in line with the logic you are trying to explain also Kherson, Zaporizhzhya, Donetsk and Luhansk areas of Ukraine are largely controlled by the Russian combatants now (formally, Russia applied the same approach as for Crimea there and according to the constitution they belong to the failed state). The "now" would significantly differ from hour-to-hour. Therefore, it would make a suggestion to bring Crimea back to Ukraine until it is all settled to not violate the logic and, which is more important, the international agreements / law.

Please understand that Natural Earth's mapping data is not the place to virtue signal, it simply attempts to reflect the actual condition of the territories. At best youre arguing to have all of Ukraine labelled as disputed territory

**Comment by asiryk-888** [comment]
> 

Yes, and that's precisely why it is crucial for the territory to be returned to Ukraine in the maps. Failure to do so is creating a clear inconsistency in the approach. I have already reported this matter to Ukrainian foreign policy officers, who are responsible for dealing with such cases officially. It's important to recognize that numerous individuals and organizations rely on these maps, and by allowing Russia's unlawful actions to be reflected on them, it effectively legitimizes their international crimes.

**Comment by megamster** [comment]
@asiryk-888 maps are generally supposed to reflect the reality on the ground. The reality is that Russia holds Crimea and several other part of Ukraine are currently disputed territory.

There are dozens of other cases around the world were basically everyone considers a territory part of one country but some other country has de facto control of it. Your attitude is obnoxious and offensive to everyone living in a country that is affected by a similar situation or anyone who simply had a decent education and is aware that maps are supposed to be de facto and not de jure idealized representations

**Comment by asiryk-888** [comment]
> @asiryk-888 maps are generally supposed to reflect the reality on the ground. The reality is that Russia holds Crimea and several other part of Ukraine are currently disputed territory.
> 
> There are dozens of other cases around the world were basically everyone considers a territory part of one country but some other country has de facto control of it. Your attitude is obnoxious and offensive to everyone living in a country that is affected by a similar situation or anyone who simply had a decent education and is aware that maps are supposed to be de facto and not de jure idealized representations

To "reflect the reality on the ground" is to do it on a hourly basis (also impacting the territory of Russia in Belgorod, which not controlled by the authorities). Is this what you plan to do? That is exactly why, till it is clear Crimea must be Ukrainian.

**Comment by megamster** [comment]
> > @asiryk-888 maps are generally supposed to reflect the reality on the ground. The reality is that Russia holds Crimea and several other part of Ukraine are currently disputed territory.
> > There are dozens of other cases around the world were basically everyone considers a territory part of one country but some other country has de facto control of it. Your attitude is obnoxious and offensive to everyone living in a country that is affected by a similar situation or anyone who simply had a decent education and is aware that maps are supposed to be de facto and not de jure idealized representations
> 
> To "reflect the reality on the ground" is to do it on a hourly basis (also impacting the territory of Russia in Belgorod, which not controlled by the authorities). Is this what you plan to do? That is exactly why, till it is clear Crimea must be Ukrainian.

Its precisely because wars exist in the world and territorial control during them can change quickly that territories can be labelled as disputed...

**Comment by asiryk-888** [comment]
> > > @asiryk-888 maps are generally supposed to reflect the reality on the ground. The reality is that Russia holds Crimea and several other part of Ukraine are currently disputed territory.
> > > There are dozens of other cases around the world were basically everyone considers a territory part of one country but some other country has de facto control of it. Your attitude is obnoxious and offensive to everyone living in a country that is affected by a similar situation or anyone who simply had a decent education and is aware that maps are supposed to be de facto and not de jure idealized representations
> > 
> > 
> > To "reflect the reality on the ground" is to do it on a hourly basis (also impacting the territory of Russia in Belgorod, which not controlled by the authorities). Is this what you plan to do? That is exactly why, till it is clear Crimea must be Ukrainian.
> 
> Its precisely because wars exist in the world and territorial control during them can change quickly that territories can be labelled as disputed...

...and this is why Crimea is part of Ukraine, which can be put on the layer of disputed territories and not vice-versa. This would be at least logical and consistent with the international agreements.

**Comment by megamster** [comment]
@asiryk-888 not at all. Crimea is de facto Russian. It would be the other Ukrainian regions that could be labelled as you propose

**Comment by asiryk-888** [comment]
> @asiryk-888 not at all. Crimea is de facto Russian. It would be the other Ukrainian regions that could be labelled as you propose

not at all. The Russian are trying to bring it under control, but most likely will not succeed in this.

**Comment by asiryk-888** [comment]
in the Russian erotic fantasies, Alaska is also Russian. Would you mark it as dispute?

**Comment by megamster** [comment]
@asiryk-888 literally every single reputable news outlet reports that Crimea continues to be under Russian control since 2014. That's all that matters when it comes to having accurate mapping data

**Comment by asiryk-888** [comment]
> @asiryk-888 literally every single reputable news outlet reports that Crimea continues to be under Russian control since 2014. That's all that matters when it comes to having accurate mapping data

I don't see any reputable source except the UN. In the UN classification, Crimea is controlled by Ukraine. This is not disputed even by the allies of Russia (e.g. China).

**Comment by megamster** [comment]
> in the Russian erotic fantasies, Alaska is also Russian. Would you mark it as dispute?

@asiryk-888 you clearly have not yet understood what de facto. As opposed to de jure means. For a de jute classification what any country says about any particular territory is irrelevant. What matters is who effectively controls it and if its unclear or subject to go back and forth then its disputed

**Comment by asiryk-888** [comment]
> > in the Russian erotic fantasies, Alaska is also Russian. Would you mark it as dispute?
> 
> @asiryk-888 you clearly have not yet understood what de facto. As opposed to de jure means. For a de jute classification what any country says about any particular territory is irrelevant. What matters is who effectively controls it and if its unclear or subject to go back and forth then its disputed

I have argued that you don't. Also make the numerous example why it is not legal, not logical and not consistant to belong Crimea to Russia.

**Comment by megamster** [comment]
> > @asiryk-888 literally every single reputable news outlet reports that Crimea continues to be under Russian control since 2014. That's all that matters when it comes to having accurate mapping data
> 
> I don't see any reputable source except the UN. In the UN classification, Crimea is controlled by Ukraine. This is not disputed even by the allies of Russia (e.g. China).

@asiryk-888 the UN is not relevant here. The UN would be relevant if discussing a de jure status, not a de facto one

**Comment by asiryk-888** [comment]
> > > @asiryk-888 literally every single reputable news outlet reports that Crimea continues to be under Russian control since 2014. That's all that matters when it comes to having accurate mapping data
> > 
> > 
> > I don't see any reputable source except the UN. In the UN classification, Crimea is controlled by Ukraine. This is not disputed even by the allies of Russia (e.g. China).
> 
> @asiryk-888 the UN is not relevant here. The UN would be relevant if discussing a de jure status, not a de facto one

you got the advice to track it hourly map of Russia to reflect de-facto. If not possible, you must return to other best option. And this one is to show the correct territory of the countries, not the imaginary ones.

**Comment by asiryk-888** [comment]
This would be a good source to check is at least daily: https://deepstatemap.live/

This map will also allow you to represent the reality of the territories, which are not controlled by Russia in the UN recognized borders of Russia. There are some minor spots near Belgorod now. But likehood that there will be much more in the next weeks is high.

**Comment by megamster** [comment]
The other best option is to mark it as disputed. Its why such label exists, as I've mention a thousand times already.

Also, thanks for making me laugh by saying it would be the literally imaginary representation of yours that would be the real one 😄

**Comment by asiryk-888** [comment]
> The other best option is to mark it as disputed. Its why such label exists, as I've mention a thousand times already.
> 
> Also, thanks for making me laugh by saying it would be the literally imaginary representation of yours that would be the real one 😄

this is exactly my suggestion - there is a layer of disputed territories. Crimea is Ukraine and the disputed status can be attributed because of the Russian claims (you can also mark Alaska in the same way, just get paid by FSB of the Russian Federation for this). The benefit is in the logic, consistency and legitimacy of the approach. An additional benefit is mitigation of the risks for the developers to be included in the sanctions lists.  Unless you live in Russia, China or some other territories having no plans for doing business anywhere else, this is also an important factor to consider.

(we are discussing just a map, but due to a good job the developers are doing, the consequents are reaching far beyond; that is why I require to have the standard map with the Ukrainian Crimea, not vice-versus)

**Comment by megamster** [comment]
@asiryk-888 again, you continue to fundamentally misunderstand what a de facto representation is supposed to be: apolitical. You're trying to politicize something that intends to be factual. and nothing else. Let's clear up some things:

- that anyone knows of there are no Russian forces controlling Alaska so it has nothing to do with this.

- by basing arguments on legitimacy having to do not with accuracy but which side of a conflict one is on, you are no better than the Russians you're supposedly against. The people doing the most harm to the Ukrainian cause are the ones trying to impose Stalinist views on others, like you are here. I can tell you that people here in Portugal have come to nowadays see the Ukrainians as little better than the Russians exactly because of such attitudes that reveal Ukrainians are still stuck in a soviet view of the world where they see everything through political lenses. Guess what, a map is just a map and people in developed countries don't take it kindly if you try to transform it into some ideal view of the world, even if they politically agree with you.

- the whole point of having a de facto policy is to be objective and not get involved in political matters. Here in Portugal, the government forbade the use of Google maps after public outcry because Google had Crimea as part of Ukraine. And not, its not because people here are some Russian puppets as your soviet view of the world would have it. Its because a map is supposed to be a map and be accurate. If anything, doing what you propose is what would get this project sanctioned in developed countries that behave as such

**Comment by asiryk-888** [comment]
> @asiryk-888 again, you continue to fundamentally misunderstand what a de facto representation is supposed to be: apolitical. You're trying to politicize something that intends to be factual. and nothing else. Let's clear up some things:
> 
> * that anyone knows of there are no Russian forces controlling Alaska so it has nothing to do with this.
> * by basing arguments on legitimacy having to do not with accuracy but which side of a conflict one is on, you are no better than the Russians you're supposedly against. The people doing the most harm to the Ukrainian cause are the ones trying to impose Stalinist views on others, like you are here. I can tell you that people here in Portugal have come to nowadays see the Ukrainians as little better than the Russians exactly because of such attitudes that reveal Ukrainians are still stuck in a soviet view of the world where they see everything through political lenses. Guess what, a map is just a map and people in developed countries don't take it kindly if you try to transform it into some ideal view of the world, even if they politically agree with you.
> * the whole point of having a de jure policy is to be objective and not get involved in political matters.

I see some limited intellectual capabilities on your side if you still follow the "Alaska case". I gave all my comments. In any case, the information for the problems was submitted to the authorities in Ukraine, the US, the EU and the UK. So, I don't care if you would understand anything from the discussion or will be forced by law.

(think of this like of being part of future genocide case - you are full informed about the consequences of your approach and cannot remain reckless)

**Comment by megamster** [comment]
@asiryk-888 a guy that clearly hasn't learn at school how maps are supposed to represent the world calling someone else intellectually challenged is bound to he the height of a lack of self awareness 😂🤣🤣

Again, in developed countries its maps that don't follow a de facto rule that are being banned from government use and the EU and several European countries have already warned Ukraine about such Stalinist views and that they are harmful to Ukraine if it wants to be seen as a Western democratic country.

**Comment by asiryk-888** [comment]
> @asiryk-888 a guy that clearly hasn't Leary at school how maps are supposed to represent the world calling someone else intellectually challenged is boung to he the height of a lack of self awareness 😂🤣🤣
> 
> Again, I'm developed countries its maps that don't follow a de facto rule that are being banned fr government use and the EU and several European countries have already warned Ukraine about such Stalinist views and that they are harmful yo Ukraine if it wants to be seen as a Western democratic country

if you want in future to be able to travel to the democracy countries or to do a business with them, you must also share some of the values. Apparently, you are enjoying opportunities, provided by the democraties in the dimensions of making international impact and speaking freely. Therefore, this must be not used in harm for all others.

From "Stalinist" quote, should I assume that you are coming from "Russia"? or you "were born in USSR"?

**Comment by megamster** [comment]
@asiryk-888  Indeed. That's what I've been telling you 😉. Democratic countries don't sanction people because of their political views or how they draw maps. What they do do is have their government cease their own use of maps that aren't accurate but instead try to virtue signal, as I told you happened here in Portugal with the banning fr government of google maps. Also, I'm Portuguese, I already have one of the strongest passports of the world and can travel to over 30 countries without even needing a passport. Its Ukrainians that should worry about the fact that stances such as yours here are making them come off as remnants of a soviet past. The very idea that you reporting anyone to the government would make that government take action against those people you reported is straight out of the soviet union, not western democracies.

**Comment by asiryk-888** [comment]
> Indeed. That's what I've been tell you 😉. Democratic countries don't sanction people because of their political views or how they draw maps. What they do do is have their government cease their own use of maps that aren't accurate but instead try to virtue signal, as I told you happened here in Portugal with the banning fr government of google maps. Also, I'm Portuguese, I already have one of the strongest passports of the world and can travel to over 30 countries without even needing a passport. Its Ukrainians that should worry about the fact that stances such as yours here are making them come off as remnants of a soviet past. The very idea that you reporting anyone to the government would make that government take action against those people you reported is straight out of the soviet union, not western democracies.

can because you are intendently provoking mass killing of people. How many European countries have recognized it as genocide already? Do you understand the legal consequences for yourself?

**Comment by megamster** [comment]
@asiryk-888 you continue to harm the cause you're pretending to defend. At the very core of democracy is liberty, and that includes liberty to disagree with whatever any government says. Obviously there aren't any sort of legal consequences for that. If you don't understand that and make ridiculous Soviet stryle claims about a map killing people by being factual instead of fictional, then you're then one contributing to to diminish the support for the Ukrainian cause. As I've already said, people see such attitudes and immediately think the Ukrainian are just as brainwashed as the Russians

**Comment by asiryk-888** [comment]
> 

you are fully confused with the definitions. And you are warned about the crime the map is performing. There is nothing to add, the officials must act now.
Regarding the legal consequences, you should better study the history of Germany and the participants of the crimes are caught by the legal system till this moment. The same case you can to follow in terms of "freedom" - if ends with the fundamental rights of others, in the way that even to draw certain symbols are not allowed.

I don't know anything about brainwashing of the Russians or Ukrainians and how did you get to this conclusions. Was this another RPG you are trying to make a map for?

**Comment by megamster** [comment]
@asiryk-888 I'll be waiting for them 😂. Congrats in corroborating the reports that have started to come out about the Ukrainian government and how repressive and brainwashing of its population it is. The Ukrainian minister of foreign affairs can come here many times like he came a couple days ago and tell people that Ukraine is a democracy and it's people aren't brainwashed but no one will believe him if people like you continue to show Ukraine is just another dictatorship with a population that thinks that I'm democratic countries governments act against people for their opinions and that its acceptable to try to politically weaponize anything and everything including maps.

You don't seem to even realize how ridiculous it sounds for you to think anyone gets sanctioned for their policy when developing mapping data or for you to think any authority will to more than laugh at whatever complaint you submit about it

**Comment by asiryk-888** [comment]
> @asiryk-888 I'll be waiting for them 😂. Congrats in corroborating the reports that have started to come out about the Ukrainian government and how repressive and brainwashing of its population it is. The Ukrainian minister of foreign affairs can come here many times like he came a couple days ago and tell people that Ukraine is a democracy and it's people aren't brainwashed but no one will believe him if people like you continue to show Ukraine is just another dictatorship with a population that thinks that I'm democratic countries governments act against people for their opinions

so, you are one of the "brainwashed" Ukrainians? :-)))) Otherwise, I would not wait for the Ukrainian officials. The laws in the US, the EU, the UK, Canada, Australia are enforced by the local regulators.

If somebody from this Ukrainian services is  reading, they must then search for you - you must be at least informed about OSINT and might be helpful for them.

**Comment by megamster** [comment]
As I've mentioned, I'm Portuguese. I can inform you that laws here are enforced by courts, the national guard and the police. Secret agencies aren't allowed to interact in any way with people outside government. They wouldn't be allowed to "search" for me or anyone else. Only the police, with a court order and only from 7am to 9pm. 😉 

That's how it works here. Its a democracy, not an Ukrainian style dictatorship. Even then, since you don't mention having submitted a lawsuit against me in Portugal, I think I'll be OK 😁

**Comment by asiryk-888** [comment]
> 

thank you for informing. So, you are safe and in comfort, enjoying sun and probably some good wine. You just need to know that to support a genocide is not a good thing and responsibility might come depending on your role. Your personal position and arguments sound strange for the situation, when Portugal is doing its best to make the things right in this situation.

The early response is more institutional as it impacts the maps that people are using by default and not knowing the situation they might put a wrong information on their visual materials. In this part it might appear that somebody purposely made the changes inconsistent with the general approach for the disputed territories. The latter may also allow to discover connections to the special services of Russia.

With all those smiles, it seems to me that you have no idea about the discussion. "Brainwashing", "Freedom of speech".... This is not a game, people are killed by hundreds and thousands on a daily basis, the total death might be close to 1 million by now, the land is ruined and will not be inhabitant because of the pollution (mines, metals and others). This is all because somebody didn't stop playing games timely. The wrong things are killing people now, the correct map is a very small correction that can result in less lost lives in future.

**Comment by megamster** [comment]
@asiryk-888 Portugal has vetoed within the EU measures that would restrict the broadcast of Russian channels and websites. So did other countries. It was also one of the countries warning Ukraine that demands for such bans were not in alignment with its claims of being a democracy. 

That of course has nothing to do with actually supporting the Ukrainian war effort, although attitudes like yours, which demonstrate that Ukrainian people don't share fundamental western values diminish the popular support for Ukraine and in turn restrict the amount of help that can be provided.

Its people like you that severely restrict the help that can be sent and thus lead to the death of many that could otherwise be saved, not a map being accurate. Even on that point, if people hear about Russia being at war with Ukraine and then the maps show Ukraine with the same area it had before, that undermines the message that there's an actual war going on.

Understand that people here have been educated to actually look at all the facts and reach their own conclusions. If they smell propaganda or manipulation, like showing a map that doesn't represent on the ground control, they turn against those who are trying to fool them. They turn against you... You're the one trying to play games by having fake maps. Free speech and rational decision is not a game, propaganda and brainwashing as you propose is

**Comment by asiryk-888** [comment]
Drink your wine, relax, watch Ukrainians being cool and real human beings making history. You will have an honor later to say that paid taxed to finance supply of tanks, cannons and armored vehicles. BTW - have you already being able to see F-16 trainings provided by Portugal?

**Comment by megamster** [comment]
Can't say I have, but they were planning to send 15 planes and now it will be much less, if any. Its an important relationship because Portugal is the only country other than the USA that has the ability to make some of the deeper repairs and upgrades that an f-16 might need. Why won't there be 15 planes? Because people like you talk nonsense about maps and Ukrainian kids at school and their parents make obnoxious complaints about the kids talking about Russia in history classes and people are tuning off from the whole situation due to all the nonsense. 

On that point, I have a relative who is a teacher and handles disciplinary complaints about other school personnel. There was an ukrainian kid and his mother who complained about a teacher and were surprised that the teacher wasn't fired and instead the kid was reprehended for making nonsense complaints. Reminds me of your strange hopes that somehow I would face consequences for not agreeing with your nonsense.

You people should really moderate yourselves and learn some western values. This is good advice, trust me

**Comment by asiryk-888** [comment]
> Can't say I have, but they were planning to send 15 planes and now it will be much less, if any. Its an important relationship because Portugal is the only country other than the USA that has the ability to make some of the deeper repairs and upgrades that an f-16 might need. Why won't there be 15 planes? Because people like you talk nonsense about maps and Ukrainian kids at school and their parents make obnoxious complaints about the kids talking about Russia in history classes and people are tuning off from the whole situation due to all the nonsense.
> 
> On that point, I have a relative who is a teacher and handles disciplinary complaints about other school personnel. There was an ukrainian kid and his mother who complained about a teacher and were surprised that the teacher wasn't fired and instead the kid was reprehended for making nonsense complaints. Reminds me of your strange hopes that somehow I would face consequences for not agreeing with your nonsense.
> 
> You people should really moderate yourselves and learn some western values. This is good advice, trust me

The good thing is that your opinion and guesses do not matter - it seems that for both, the Portuguese government and the Ukrainians. I heard that Portugal is only participating in trainings, the planes are coming from other European sources and potentially US.

Regarding the "western" values, you seems to reflect the wrong ones - the colonial times are ended now and will not come back, you also must start value people lifes. You can do your best in supporting the Russian colonial effort, but it doesn't really matter .

**Comment by megamster** [comment]
@asiryk-888 the initial plan was to send 15, it was turned into zero after it got out and people complained. Obviously it wasn't just me complaining, I wasn't even one of those who protested. Again, we are a democracy, the government does what people want it to do, within reason. The Ukrainian minister of Foreign Affairs was here a couple days ago, as I've already said, and made a speech basically saying you people aren't the autocratic pests you come out as, in hopes of securing some of those planes back. Maybe he will get 2 or 3, never 15 as initially planned.

You really don't seem to understand that talking about maps having to be "correct" and things in general having to adhere to some kind of legal fiction is exactly what people hear about dictatorships and how they work. Its the worst type of message you can send

**Comment by asiryk-888** [comment]
> @asiryk-888 the initial plan was to send 15, it was turned into zero after it got out and people complained. Obviously it wasn't just me complaining, I wasn't even one of those who protested. Again, we are a democracy, the government does what people want it to do, within reason. The Ukrainian minister of Foreign Affairs was here a couple days ago, as I've already said, and made a speech basically saying you people aren't the autocratic pests you come out as, in hopes of securing some of those planes back. Maybe he will get 2 or 3, never 15 as initially planned.
> 
> You really don't seem to understand that talking about maps having to be "correct" and things in general having to adhere to some kind of legal fiction is exactly what people hear about dictatorships and how they work. Its the worst type of message you can send

as advised, check the German experience and consider you personal role in the mass killings of civilians, which you promote and actively support ("stalins", "brainwashings", the way you expect you personal role in the crimes should be tolerated etc). You can avoid the effective criminal responsibility (99,9% you would in this point) as this might be too small and too mean to follow you personally. At the same time as a tool / map, impacting decisions of the people making crimes it is already reported (the way the Russians can legalize their crimes or the way you were incentified to mis-report the world map in the aggressor's interest).

It is not about the political systems, but about lives. The Ukrainian lives matter for me. For yourself you can pick up any explanation - finally, in some areas of Amazonia people are just another type of food. It is not possible to explain to cannibal any other values, as it can be always argued that those are real  "western values" as Amazonia is located further west than Portugal and freedom of eating must be secured.

**Comment by megamster** [comment]
My ancestors died to achieve freedom. Saving lives is the shambolic argument and excuse of every would be fascist or communist dictator in orders to subdue others. 

My country by remaining neutral during the second world war saved more lives than if it had participated in that open air massacre. Congrats in destroying your argument to politicize a simple mapping tool 😉

Also, let me be clear that I had no intervention in how Natural Earth assigns disputed Ukrainian territory or formerly Ukrainian territory.

**Comment by asiryk-888** [comment]
it sounds like your ancestors were great people. This puts even more shame on you - because you spoil their heritage supporting new Nazis.
Again, you seems to be not clever at all missing the argument - it is going far beyond mapping. It is a free licensed that can be picked up by many others spreading the crimes and supporting their acceptance. In any ethical code this would be found "reckless". What is play and drunk discussion for you are lives for others, this is not tolerable. Your ancestors would most likely understood this, even choosing to stand aside and watch evil rather than fight it. You go further and support.

**Comment by megamster** [comment]
@asiryk-888 it continues being funny how the only thing you've done is make a fool of yourself and still somehow haven't realized you're the one coming across as a not very intellectually brilliant, to put it mildly.

It goes without saying that no actual ethical code sanctions lying in order to serve political purposes, not mater how great they might be. Its reckless and intellectually dishonest to deny reality and try to impose such distorted view on other for propagandistic purposes.

Please understand that viewing everything in the world by its value in pursuing some political aim is in itself an authoritarian and  dictatorial stance. Western democracies don't do that. Its places like the Soviet Union and Russia that do that, coupling it with the emotional blackmail you're also so keen on putting forth. It doesn't work with people educated in developed and free countries and looks outright foolish and third world. That's how you look. 

People aren't Russian supporters by wanting an accurate map. They just want an accurate map. Maps are tools, not devices for political propaganda

**Comment by asiryk-888** [comment]
> @asiryk-888 it continues being funny how the only thing you've done is make a fool of yourself and still somehow haven't realized you're the one coming across as a not very intellectually brilliant, to put it mildly.
> 
> It goes without saying that no actual ethical code sanctions lying in order to serve political purposes, not mater how great they might be. Its reckless and intellectually dishonest to deny reality and try to impose such distorted view on other for propagandistic purposes.
> 
> Please understand that viewing everything in the world by its value in pursuing some political aim is in itself an authoritarian and dictatorial stance. Western democracies don't do that. Its places like the Soviet Union and Russia that do that, coupling it with the emotional blackmail you're also so keen on putting forth. It doesn't work with people educated in developed and free countries and looks outright foolish and third world. That's how you look.
> 
> People aren't Russian supporters by wanting an accurate map. They just want an accurate map. Maps are tools, not devices for political propaganda

You have all arguments, just try to catch up with them and make the correct map.

**Comment by megamster** [comment]
@asiryk-888 there you go again, not an accurate map but a "correct" one 😉. 

Also, let's be clear that I have no involvement with the Natural Earth project. I might contribute like anyone else but Im in no way responsible for it

**Comment by nvkelso** [comment]
The discussion here has veered into uncharted territories, as it were. I'm locking comments on this issue for 6 months and will start blocking GitHub users who can't maintain decorum.
